### ISSUE TITLE
[18Rhl] - Incorrect tile upgrade

### ISSUE THREAD

**Issue by ddaybell** [tobymao/18xx#7113]
https://18xx.games/game/72137#map

The game allows me to upgrade Duisberg, Dusseldorf and Koln to tile #950, which is a 30-Mark Moens tile.  This seems wrong.  It should upgrade to the #932V tile, I believe. 

Don

**Comment by perwestling** [comment]
Yes, this is a bug in the Promotion tile variant. It was reported in #7070, and will be corrected shortly. I am closing this one as it is a duplicate.
